macros/contexts/contextBoolean.pl

```diff
@@ -0,0 +1,253 @@
+	$context->{parser}{Number}  = 'context::Boolean::Boolean';
+	$context->{value}{Boolean}  = 'context::Boolean::Value';
+	$context->{value}{Real}     = 'context::Boolean::Value';
+
+	$context->{pattern}{number}       = '^[01TtFf]$';
+	$context->{pattern}{signedNumber} = '^[01TtFf]$';
+	$context->flags->set(
+		NumberCheck => sub {
+			my $self = shift;
+			$self->{value}  = ($self->{value_string} =~ /[t1]/i) ? 1 : 0;
+			$self->{isOne}  = ($self->{value} == 1);
+			$self->{isZero} = ($self->{value} == 0);
+		}
+	);
+	$context->update;
+
+	my $orPrec  = $context->operators->get('+')->{precedence};
+	my $andPrec = $context->operators->get('*')->{precedence};
+	my $notPrec = $context->operators->get('u-')->{precedence};
+
+	## Disable unnecessary context stuff
+	$context->functions->clear();
+	$context->strings->clear();
+	$context->lists->clear();
+
+	## Define constants for 'True' and 'False'
+	$context->constants->are(
+		'T' =>
+			{ value => context::Boolean::Value->new(1), isConstant => 0, string => 'T', TeX => '\top', perl => 1 },
+		'F' =>
+			{ value => context::Boolean::Value->new(0), isConstant => 0, string => 'F', TeX => '\bot', perl => 0 },
+		# 0 and 1 are not recognized unless added as constants
+		# it might not be a bad idea anyhow?
+		'1'     => { alias => 'T' },
+		'0'     => { alias => 'F' },
+		'True'  => { alias => 'T' },
+		'False' => { alias => 'F' },
+	);
+
+	## Define our logic operators
+	$context->operators->are(
+		'or' => {
+			class         => 'context::Boolean::BOP::or',
+			precedence    => $orPrec,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' or ',
+			TeX           => '\vee ',
+			perl          => '||',
+			alteratives   => ['\x{2228}'],
+		},
+		'and' => {
+			class         => 'context::Boolean::BOP::and',
+			precedence    => $andPrec,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' and ',
+			TeX           => '\wedge ',
+			perl          => '&&',
+			alteratives   => ['\x{2229}'],
+		},
+		'xor' => {
+			class         => 'context::Boolean::BOP::xor',
+			precedence    => $andPrec,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' xor ',
+			perl          => '!=',
+			TeX           => '\oplus ',
+			alteratives   => [ '\x{22BB}', '\x{2295}' ],
+		},
+		# unary negation is only allowed if the '-' token is recognized, so make it BOP::undef
+		'-' => {
+			class         => 'Parser::BOP::undefined',
+			precedence    => $notPrec - 1,
+			associativity => 'left',
+			type          => 'both',
+			string        => '-',
+		},
+		'not' => {
+			class            => 'context::Boolean::UOP::not',
+			precedence       => $notPrec,
+			associativity    => 'left',
+			type             => 'unary',
+			string           => ' not ',
+			TeX              => '\mathord{\sim}',
+			perl             => '!',
+			alteratives      => [ '\x{172}', '\x{732}' ],
+			nofractionparens => 1,
+		},
+		' '  => { alias => 'and' },
```

Comment:
I forgot to add the `precedence => $andPrec` and other properties (`associativity` and `type`).

Reply:
to clarify, the empty string token must include precedence, associativity, and type?

Implicit multiplication appears to be working without these changes... 

If I'm reading the code (Parser.pm sub Op) correctly, adding a precedence value would be necessary in order to ensure that operations of higher precedence are resolved first (in this case there aren't any -- still, not a reason to do Bad Things™️), then without associativity or type, we end up falling through all the other logic right into where we need to be.

Reply:
Yes, this was to get the order of operations correct.  When I made the comment, the precedences were different, and that meant that implicit multiplication didn't have the correct precedence, so `p or q p` would be `(p or q) and p` rather than `p or (q and p)`.  I don't remember what the default precedence is, so I would still set that for `' '` to be sure.